components/Author/Tabs/FeedCard.tsx

```diff
@@ -285,6 +273,17 @@ function FeedCard({
   const bountyAmount = documentFilter?.bounty_total_amount;
   const hasActiveBounty = documentFilter?.bounty_open;
 
+  let numOfVisibleHubs = 3;
```

Comment:
Ideally this display logic would be done via CSS but given how the component is structured, it is not possible to do so without rewriting or relying on JS.